plugins/copr.py

```diff
@@ -401,16 +401,18 @@ def _guess_chroot(chroot_config):
         if dist is None or (dist[0] is False) or (dist[1] is False):
             dist = linux_distribution()
         if "Fedora" in dist:
-            # x86_64 because repo-file is same for all arch
-            # ($basearch is used)
+            distarch = rpm.expandMacro("%{_arch}")
```

Comment:
This is wrong. You should as DNF about that architecture. Try this script:
```
#!/usr/bin/python3
import dnf
b = dnf.Base()
print(b.conf.substitutions) 
```
the `b` is available as self.base in the plugin.